edx_when/models.py

```diff
@@ -46,12 +50,12 @@ def actual_date(self, schedule=None):
         """
         if self.rel_date is not None:
             if schedule is None:
-                raise ValueError(
-                    "Can't interpret relative date {} for {!r} without a user schedule".format(
-                        self.rel_date,
-                        self
-                    )
+                LOG.warning(
+                    "Can't interpret relative date %s for %r without a user schedule",
+                    self.rel_date,
+                    self,
                 )
+                return None
```

Comment:
Is this the first time this method can return `None` vs throwing an exception? If so, I'd say that's an API change that deserves a major version bump, as it would require client code changes to handle.

Reply:
Good question. It could return `None` if both `abs_date` and `rel_date` were `None`, but I'm not sure if that could actually happen in existing code paths. I'll poke around and see if there are any callers that will be tripped up.

Reply:
Ok, I reverted that change to the api in favor of just not adding those dates to the list of returned dates when you load from a course.